scipy/fft/tests/test_real_transforms.py

```diff
@@ -241,7 +243,16 @@ def test_orthogonalize_dcst3(func, norm, xp):
 
 @skip_xp_backends(np_only=True,
                   reason='array-likes only supported for NumPy backend')
-@pytest.mark.parametrize("func", [dct, idct, dctn, idctn, dst, idst, dstn, idstn])
```

Comment:
I think we should just remove the `xp` fixture here to save ten lines

Reply:
Thanks, good suggestion. I removed `xp` from another test in this file too.